security/collector.go

```diff
@@ -0,0 +1,108 @@
+package security
+
+import (
+	"encoding/xml"
+	"strings"
+
+	"github.com/czerwonk/junos_exporter/collector"
+	"github.com/czerwonk/junos_exporter/rpc"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const prefix string = "junos_security_"
+
+var (
+	fpcNumber          *prometheus.Desc
+	picNumber          *prometheus.Desc
+	cpuUtilization     *prometheus.Desc
+	memoryUtilization  *prometheus.Desc
+	currentFlowSession *prometheus.Desc
+	maxFlowSession     *prometheus.Desc
+	currentCpSession   *prometheus.Desc
+	maxCpSession       *prometheus.Desc
+)
+
+func init() {
+	l := []string{"target", "re_name"}
+	//l := []string{"target", "re_name", "description", "name"}
```

Comment:
please remove comments when not needed any more

Reply:
I've removed the comment.